chpl/chpl-service/pom.xml

```diff
@@ -470,7 +470,6 @@
                         <source>src/main/java/gov/healthit/chpl/domain/CertifiedProductQmsStandard.java</source>
                         <source>src/main/java/gov/healthit/chpl/domain/CertifiedProductSearchDetails.java</source>
                         <source>src/main/java/gov/healthit/chpl/domain/CertifiedProductTargetedUser.java</source>
-                        <source>src/main/java/gov/healthit/chpl/domain/Contact.java</source>
```

Comment:
Does the new domain object "PointOfContact" need to be added here?

Reply:
No, and I don't understand why. I originally did add them but the mvn task that creates the XSD failed with errors. Removing the objects from here allowed the XSD to get created AND it has the new objects. I could not figure out why.